lib/installation-proxy/index.js

```diff
@@ -0,0 +1,56 @@
+const INSTALLATION_PROXY_SERVICE_NAME = 'com.apple.mobile.installation_proxy';
+
+class InstallationProxyService {
+  constructor (plistService) {
+    this.plistService = plistService;
+  }
+
+  async listApplications (applicationType = 'User') {
+    const request = {
+      Command: 'Browse',
+      ClientOptions: {
+        ApplicationType: applicationType
+      }
+    };
+
+    this.plistService.sendPlist(request);
```

Comment:
this shouldn't be awaited, right?

Reply:
This is not async. Just passing an object to the `encoder.write`